rel/overlay/etc/default.ini

```diff
@@ -147,6 +147,27 @@ max_db_number_for_dbs_info_req = 100
 ; prevent non-admins from accessing /_all_dbs
 ; admin_only_all_dbs = true
 
+;;;; Moved from [httpd] ;;;;;
+;secure_rewrites = true
+;allow_jsonp = false
+
+;enable_cors = false
+;enable_xframe_options = false
+
+; CouchDB can optionally enforce a maximum uri length;
+; max_uri_length = 8000
```

Comment:
It might be better to be consistent and use the format where there is no space between ';' and the configuration key but there is a space between ';` and comment line. For example:

```
; This is a comment
;this_is_a_config_key = foo
```

The reason to do that is it makes the comment and the keys stand out a bit.